packages/headless/src/controllers/commerce/context/cart/headless-cart.ssr.ts

```diff
@@ -0,0 +1,23 @@
+import {SharedControllerDefinitionWithoutProps} from '../../../../app/commerce-ssr-engine/types/common';
+import {Cart, buildCart, CartProps, CartInitialState} from './headless-cart';
+
+export type {CartState, CartItem, CartProps} from './headless-cart';
+export type {Cart, CartInitialState};
+
+export interface CartDefinition
+  extends SharedControllerDefinitionWithoutProps<Cart> {}
+
+/**
+ * Defines a `Cart` controller instance.
```

Comment:
In general we're not really defining an "instance".

Maybe we could change those docs strings to

"Defines the `MyControllerName` controller for the purpose of server-side rendering."

Reply:
did it for all my PRs.